### PR TITLE
Dev v2.1.0

### PR DESCRIPTION
**Pull request by ronaldddilley**
Enhancements for extents
Enhancements for antimeridian

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 
-This project contains application source code for extracting elevation data from external sources and generating an elevation product conforming to the input data.
+This project contains application source code for extracting elevation data from external sources and generating an elevation product conforming to the input data. An option exists to specify a user-defined geographic extents (versus the default scene extents) to produce elevation for a different footprint from the scene.
 
 ## Release Notes
-* Initial project version
-* Migrated from original source location to this project
-* Minor changes for collection based processing
+* Supported optional user-defined geographic extents
+* Supported optional user-defined output filename
```
Comment (ronaldddilley) [inline README.md]: Ray, Would you please add a note or two about support for the antimeridian to the release notes.

**Comment by ronaldddilley** [comment]
No further comments

File: README.md (hunk 1 above)
Comment (raydittmeier) [inline README.md]: I added a line about the antemeridian fix to the release notes section.

**Comment by raydittmeier** [comment]
Leave a comment

**Comment by raydittmeier** [comment]
No further comments.

File: scripts/build_elevation_band.py [hunk 2]
```diff
@@ -386,11 +391,19 @@ def __init__(self):
...
+        # specified then use it otherwise use the default filename.
         self.elevation_format = 'ENVI'
         self.elevation_type_int16 = 'Int16'
-        self.elevation_header_name_fmt = '{0}_elevation.hdr'
-        self.elevation_image_name_fmt = '{0}_elevation.img'
+        if elevation_filename is None:
+            self.elevation_image_name_fmt = '{0}_elevation.img'
+            self.elevation_header_name_fmt = '{0}_elevation.hdr'
+        else:
+            print('DEBUG: elevation filename: {0}'.format(elevation_filename))
```
Comment (gschmidt-usgs) [inline scripts/build_elevation_band.py]: Oops.  I forgot to remove this and didn't use the logger.debug.  This DEBUG line can be removed.

**Comment by gschmidt-usgs** [comment]
Code review complete.

**Pull request merged**